eclair-core/src/test/scala/fr/acinq/eclair/blockchain/electrum/ElectrumWalletBasicSpec.scala

```diff
@@ -36,7 +37,7 @@ class ElectrumWalletBasicSpec extends AnyFunSuite with Logging {
 
   val swipeRange = 10
   val dustLimit = 546 sat
-  val feeRatePerKw = 20000
+  val feeRatePerKw = FeeratePerKw(20000 sat)
```

Comment:
Nit: names could be simplified, since there is no ambiguity on the value:

```suggestion
  val feerate = FeeratePerKw(20000 sat)
```